doc/user_guide/streak_images.rst

```diff
@@ -0,0 +1,53 @@
+.. _streak_images:
+
+Working with streak images
+**************************
+
+LumiSpy has implemented handling of streak camera images, which are a special
+subclass of :class:`hyperspy.api.signals.Signal2D` as the two signal axes have
+different units: Namely, wavelength (or energy) and time. The signal class is 
+:class:`~.signals.luminescence_transientspec.LumiTransientSpectrum`. An important
+feature is that also for streak camera images (or linescans/maps of these), the
+wavelength axis can be transformed to e.g. a :ref:`energy_axis`.
+
+.. Note::
+
+    For streak images, not all tools available for :class:`Signal2D
+    <hyperspy.api.signals.Signal2D>` will work. However, in particular
+    :external+hyperspy:ref:`peak_finding-label`
+    can be helpful to identify local maxima in the images.
+
+
+Transition to one-dimensional signals
+=====================================
+
+When transitioning from a streak image to a one-dimensional signal, LumiSpy
+uses the axes units of the new signal (whether it is time units or not) to
+assign the correct 1D signal class: Either :class:`~.signals.luminescence_spectrum.LumiSpectrum` or
+:class:`~.signals.luminescence_transient.LumiTransient`. Such a reduction of the signal dimensionality
+can be done for example by slicing the signal with
+:external+hyperspy:ref:`numpy-style indexing <signal.indexing>` or by using
+functions such as :external:meth:`sum() <hyperspy.api.signals.BaseSignal.sum>`
+and :external:meth:`integrate1D() <hyperspy.api.signals.BaseSignal.integrate1D>`
+
+In the following image, the spectrum summed over all times is obtained from the
+streak image by:
+
+.. code-block:: python
+
+    >>> s.sum(axis='Time')
+    >>> # Or alternatively:
+    >>> s.sum(axis=-1)
+
+Similarly, the transient summed over all wavelengths is obtained by:
+
+.. code-block:: python
+
+    >>> s.sum(axis='Wavelength')
+    >>> # Or alternatively:
+    >>> s.sum(axis=-2)
+
+.. image:: images/streakmap.svg
+  :width: 700
+  :alt: Plot of a streak camera image and its one-dimensional representations
```

Comment:
Is there code in lumispy that can make this figure? When I do `lumispy.signals.luminescence_transientspec.LumiTransientSpectrum.plot`, I get a normal plot.

It would be good to add the code to make this figure, otherwise it should be removed?

Reply:
I had previously prepared this figure for illustration purposes and thought it could be useful here to show the principle. It is indeed a useful feature to generally faciliate such kind of plots and I will open an issue to implement that in a separate PR (which could probably be taken on by one of my students). Not as the standard plotting of this class, but as a separate plot function (alternative to using `sum` an option to display spectra/transients for a certain range (or even an interactive span functionality can then be added later).